src/utils/logger.ts

```diff
@@ -0,0 +1,28 @@
+import { createLogger, format, transports } from 'winston';
+
+const logger = createLogger({
+  level: 'info',
+  format: format.combine(
+    format.timestamp({
+      format: 'YYYY-MM-DD HH:mm:ss',
+    }),
+    format.errors({ stack: true }),
+    format.prettyPrint(),
+    format.json()
+  ),
+  defaultMeta: { service: 'alpha-ec-be' },
+  transports: [
+    new transports.File({ filename: 'error.log', level: 'error' }),
+    new transports.File({ filename: 'combined.log' }),
+  ],
```

Comment:
add those loggs  into one folder

Reply:
I have changed this function to check if the `logs` is not available and creates it to keep all the logs  👍